deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+  description: |2
+
+    Intel PMEM-CSI is a [CSI](https://github.com/container-storage-interface/spec)
+    storage driver for container orchestrators like Kubernetes. It makes local
+    persistent memory ([PMEM](https://pmem.io/)) available as a filesystem volume to
+    container applications. It can currently utilize non-volatile memory devices
+    that can be controlled via the [libndctl utility library](https://github.com/pmem/ndctl).
+    *Persistent memory* in this context refers to a non-volatile dual in-line
+    memory module (NVDIMM). See [PMEM-CSI documentation](http://githbu.com/intel/pmem-csi)
+    for more details.
+
+    This is the operator to deploy and manage [PMEM-CSI](https://github.com/intel/pmem-csi)
+    driver on a Kubernetes cluster.
+
+    ## Prerequisite
+    ### Software required
+
+    The recommended minimum Linux kernel version for running the PMEM-CSI driver is 4.15.
+    See [Persistent Memory Programming](https://pmem.io/2018/05/15/using_persistent_memory_devices_with_the_linux_device_mapper.html)
+    for more details about supported kernel versions.
+
+    ### Hardware required
+
+    Persistent memory device(s) are required for operation. However, some development
+    and testing can be done using [QEMU-emulated](https://docs.pmem.io/persistent-memory/getting-started-guide/creating-development-environments/virtualization/qemu)
+    persistent memory devices.
+
+    ### Cluster requirements
+
+    The operator can be installed on any OLM-enabled Kubernetes cluster with version >= 1.11.
+    Make sure that the alpha feature gates _`CSINodeInfo`_ and _`CSIDriverRegistry`_ are enabled
+    in your cluster. The method to configure alpha feature gates may vary, depending on
+    the Kubernetes deployment. It may not be necessary anymore when the feature has reached
+    beta state, which depends on the Kubernetes version.
+
+    You also need to have the [kubectl CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    already configured to access your cluster.
+
+    ### Preparing worker nodes
+
+    **Persistent memory pre-provisioning**
+
+    The PMEM-CSI driver needs pre-provisioned regions on the NVDIMM device(s).
+    The PMEM-CSI driver itself intentionally leaves that to the administrator who
+    then can decide how much and how PMEM is to be used for PMEM-CSI.
+    When running the Kubernetes cluster and PMEM-CSI on bare metal, the [ipmctl](https://github.com/intel/ipmctl)
+    utility can be used to create regions.
+
+    Example of creating regions using all NVDIMMs:
+    ```
+    $ ipmctl create -goal PersistentMemoryType=AppDirect
+    ```
+    **Label the cluster nodes that provide persistent memory device(s)**
+    ```
+    $ kubectl label node <your node> storage=pmem
+    ```
+
+    ## Deploying PMEM-CSI driver
+
+    Once after installing the PMEM-CSI operator, you can deploy PMEM-CSI driver by
+    creating an instance of PMEM-CSI deployment custom resource as shown below:
```

Comment:
"the PMEM-CSI deployment custom resource"

Reply:
Done.